DigitalLearningSolutions.Web/Styles/shared/searchableElements/sort.scss

```diff
@@ -1,99 +1,81 @@
 @import "~nhsuk-frontend/packages/core/all";
 @import "common";
 
-.sort-dropdown {
-  border-radius: 4px;
+.sort-form {
+  width: 100%;
 }
 
-.sort-by-dropdown {
-  @extend .left-dropdown;
-}
-
-.sort-direction-dropdown {
-  @extend .input-with-no-js-submit-button;
+.sort-box-flex-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 100%;
 
-  width: 63%;
+  .sort-by-dropdown {
+    @extend .left-dropdown;
+    display: flex;
 
-  @include mq($until: desktop) {
-    .js-enabled & {
-      width: 70%;
+    @include mq($until: tablet) {
+      width: 100%;
     }
-  }
-
-  @include mq($from: $xl-desktop) {
-    width: 74%;
-  }
-}
 
-.sort-direction-dropdown-container {
-  @include mq($from: desktop) {
-    width: calc(59% - 12px);
-    float: left;
+    @include mq($from: tablet) {
+      flex: 1;
+    }
   }
-}
 
-.sort-submit {
-  @extend .right-submit-button;
-}
+  .sort-order-button-flex-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
 
-.sort-row {
-  display: flex;
-  flex-direction: row;
+    .search-sort-flex-container & {
+      @include mq($from: tablet) {
+        width: 50%;
+      }
+    }
 
-  @include mq($until: desktop) {
-    margin-top: 36px;
-    margin-left: 16px;
-    margin-right: 16px;
-    flex-direction: column;
+    @include mq($from: tablet) {
+      width: 60%;
+    }
 
-    .js-enabled & {
-      flex-direction: row;
+    @include mq($from: large-desktop) {
+      .js-enabled & {
+        width: 33.3333%;

```

Comment:
Should this be half of the container? (based on screenshots with JS enabled - looks a bit odd when it's not 50-50)

Reply:
I think either is probably fine. 33.33% lines it up with the buttons. 50% would make it closer to the no JS styling though where it is 50% but the button steals some of it.